checks/prometheus_check.py

```diff
@@ -29,7 +33,14 @@ class PrometheusFormat:
     PROTOBUF = "PROTOBUF"
     TEXT = "TEXT"
 
+
+class UnknownFormatError(TypeError):
+    pass
+
+
 class PrometheusCheck(AgentCheck):
+    unwanted_labels = ["le", "quantile"]  # are specifics keys for prometheus itself
```

Comment:
most of agent5 set constants as module constants, see https://github.com/DataDog/dd-agent/blob/master/utils/dockerutil.py#L44
Could you please adjust?